DatadogCrashReporting/Sources/DatadogCrashReporter.swift

```diff
@@ -7,10 +7,10 @@
 import Foundation
 import DatadogInternal
 
-/* public */ internal class CrashReporter: DatadogFeature {
-    /* public */ static let name = "crash-reporter"
+public final class DatadogCrashReporter: DatadogFeature {
+    public static let name = "crash-reporter"
 
-    /* public */ let messageReceiver: FeatureMessageReceiver
+    public let messageReceiver: FeatureMessageReceiver
```

Comment:
It has been made public because it complies with `DatadogFeature`, but that's a good call and I think we should split the interface to only expose what's necessary for the user.